src/components/ButtonPanel.js

```diff
@@ -1,46 +1,51 @@
 /* eslint-disable react/prefer-stateless-function */
 
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 import Button from './Button';
 
 
 class ButtonPanel extends Component {
   render() {
+    const { clickHandler } = this.props;
     return (
       <div className="button-panel">
         <div className="group1">
-          <Button content="AC" color="cl-light-grey" />
-          <Button content="+/-" color="cl-light-grey" />
-          <Button content="X" color="cl-light-grey" />
-          <Button content="÷" />
+          <Button handleClick={clickHandler} content="AC" color="cl-light-grey" />
+          <Button handleClick={clickHandler} content="+/-" color="cl-light-grey" />
+          <Button handleClick={clickHandler} content="X" color="cl-light-grey" />
+          <Button handleClick={clickHandler} content="÷" />
```

Comment:
I am not sure if I had mentioned this before but:

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

Reply:
Done :+1: